src/VacanciesManage/SFA.DAS.VacanciesManage/Application/Recruit/Commands/CreateVacancy/CreateVacancyCommandHandler.cs

```diff
@@ -38,6 +49,17 @@ public async Task<CreateVacancyCommandResponse> Handle(CreateVacancyCommand requ
             
             if(request.AccountIdentifier.AccountType == AccountType.Provider)
             {
+                _logger.LogDebug($"Sending details from Command Handler: UKPRN:{request.PostVacancyRequestData.User.Ukprn}");
+                _logger.LogCritical($"Sending details from Command Handler: UKPRN:{request.PostVacancyRequestData.User.Ukprn}");
+                _logger.LogError($"Sending details from Command Handler: UKPRN:{request.PostVacancyRequestData.User.Ukprn}");
```

Comment:
Can probably remove these

Reply:
obsolete code removed as not required added it for troubleshooting, but has been removed from the codebase.